### ISSUE TITLE
A11y: Form labels conflict with each other

### ISSUE THREAD

**Issue by maikmetzger** [shopware/platform#5120]
*PHP Version:* 8.2.20   | *Shopware Version:* 6.6.4.1  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

We use both, the contact form and the newsletter form on the same page.
However, as shops need to be compliant with the newest accessibility guidelines, we are really thorough with the approach.
We expected them to have different and unique labels so it won't throw accessibility errors using the WAVE accessibility checker browser addon.

*_Actual behaviour:_*

The email field label of the newsletter form and the email field label of the contact form are equal "email". This will clash with accessibility checkers. 
Should have unique label texts.

*_How to reproduce:_*

putting both on the same template/page.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38975**



[created from NEXT-38975, comment 503622]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-38975, comment 520251]
